### PR TITLE
Add more information to getBlockResponse

### PR DESCRIPTION
**Pull request by hairtail**
## Documentation

Does this change require any updates to the Iron Fish Docs (ex. [the RPC API
Reference](https://ironfish.network/docs/onboarding/rpc/chain))? If yes, link a
related documentation pull request for the website.

```
[ ] Yes
```

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: ironfish/src/rpc/routes/chain/getBlock.ts [hunk 1]
```diff
@@ -148,14 +132,31 @@ routes.register<typeof GetBlockRequestSchema, GetBlockResponse>(
     const transactions: GetBlockResponse['block']['transactions'] = []
 
     for (const tx of block.transactions) {
-      const fee = tx.fee()
-
       transactions.push({
-        signature: tx.transactionSignature().toString('hex'),
```
Comment (danield9tqh) [inline ironfish/src/rpc/routes/chain/getBlock.ts:154]: Can we add this signature field to `RpcTransaction` for better backwards compatibility of this endpoint

Comment (hairtail) [reply ironfish/src/rpc/routes/chain/getBlock.ts:154]: sure

**Review by danield9tqh — APPROVED** [review]

**Comment by danield9tqh** [comment]
@hairtail can you fix this one test that's failing? We change the response type of fee from a string to a number in this PR.
Test is here: https://github.com/iron-fish/ironfish/blob/0bb7033efe9e5408ef9d0eed693261b0bf785db9/ironfish/src/rpc/routes/chain/getBlock.test.ts#L131

I've checked in our codebase and can't find anything that depends on it being a string but we will put it in the release notes that this RPC route has changed. I think ideally it should be a string not a number since it's a bigint but we can save that migration for later down the line.

**Comment by hairtail** [comment]
> @hairtail can you fix this one test that's failing? We change the response type of fee from a string to a number in this PR. Test is here:
> 
> https://github.com/iron-fish/ironfish/blob/0bb7033efe9e5408ef9d0eed693261b0bf785db9/ironfish/src/rpc/routes/chain/getBlock.test.ts#L131
> 
> I've checked in our codebase and can't find anything that depends on it being a string but we will put it in the release notes that this RPC route has changed. I think ideally it should be a string not a number since it's a bigint but we can save that migration for later down the line.

Done

**Pull request merged**